### PR TITLE
Add Instructions Component

### PR DESCRIPTION
**Pull request by shirishgoyal**
Add static text field
Handle count for display and input fields

### CODE REVIEW

File: static/templates/template/components/instructions-edit.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ng-include src="'/static/templates/template/components/question-edit.html'"></ng-include>
```
Comment (dmorina) [inline static/templates/template/components/instructions-edit.html]: we might wanna make this a textarea

Comment (shirishgoyal) [inline static/templates/template/components/instructions-edit.html]: sure

File: crowdsourcing/serializers/task.py [hunk 2]
```diff
@@ -362,5 +363,9 @@ def get_worker_count(obj):
         return obj.task_workers.filter(status__in=[2, 3, 5]).count()
 
     @staticmethod
-    def get_completion(obj):
-        return str(obj.task_workers.filter(status__in=[2, 3, 5]).count()) + '/' + str(obj.project.repetition)
+    def get_completed(obj):
+        return obj.project.repetition
```
Comment (dmorina) [inline crowdsourcing/serializers/task.py]: did you swap these two methods?

**Comment by dmorina** [comment]
lgtm

**Pull request merged**